tutorials/using-cloud-vpn-with-cisco-asr/index.md

```diff
@@ -0,0 +1,744 @@
+    *   **Region** — The region where you want to locate the VPN gateway. Normally, this is the region that contains the instances you wish to reach. Example: `us-east1`
+    *   **IP address** — Select a pre-existing [static external IP address](https://cloud.google.com/compute/docs/ip-addresses#reservedaddress). If you don't have a static external IP address, you can create one by clicking New static IP address in the pull-down menu. Selected `vpn-scale-test0` for this guide.
+1.  Populate fields for at least one tunnel:
+    *   **Peer IP address** — Enter your peer public IP address here, with the above mentioned topology it is `204.237.220.4` 
+    *   **IKE version** — IKEv2 is preferred, but IKEv1 is supported if that is all the peer gateway can manage.
+    *   **Shared secret** — Used in establishing encryption for that tunnel. You must enter the same shared secret into both VPN gateways. If the VPN gateway device on the other side of the tunnel doesn't generate one automatically, you can make one up.
+    *   **Remote network IP range** — `10.0.0.0/8`. The range, or ranges, of the peer network, which is the network on the other side of the tunnel from the Cloud VPN gateway you are currently configuring. 
+    *   **Local subnets** — Specifies which IP ranges will be routed through the tunnel. This value cannot be changed after the tunnel is created because it is used in the IKE handshake.
+    *   Select the gateway's entire subnet in the pull-down menu. Or, you can leave it blank since the local subnet is the default.
+    *   Leave **Local IP ranges** blank except for the gateway's subnet.
+1.  Click **Create** to create the gateway and initiate all tunnels, though tunnels will not connect until you've completed the additional steps below. This step automatically creates a network-wide route and necessary forwarding rules for the tunnel.
+1.  [Configure your firewall rules](https://cloud.google.com/compute/docs/vpn/creating-vpns#configuring_firewall_rules) to allow inbound traffic from the peer network subnets, and you must configure the peer network firewall to allow inbound traffic from your Compute Engine prefixes.
+    *   Go to the [Firewall rules](https://console.cloud.google.com/networking/firewalls) page.
+    *   Click **Create firewall rule**.
+    *   Populate the following fields:
+        *   **Name:** `vpnrule1`
+        *   **VPC network:** `vpn-scale-test-cisco`
+        *   **Source filter:** IP ranges.
+        *   **Source IP ranges:** The peer ranges to accept from the peer VPN gateway.
+        *   **Allowed protocols and ports:** `tcp;udp;icmp`
+    *   Click **Create**.
+
+#### Using gcloud command-line tool
+
+1.  Create a custom VPC network. You can also use auto VPC network, make sure there is no conflict with your local network range.
```

Comment:
These are all rendering as number "1". I think you need to remove all the triple backticks as noted in the last review and instead indent all the code blocks.

Reply:
Does it look better now ? I see them rendered as 1. 2...